WeArtTrackingRawData.cpp

```diff
@@ -3,7 +3,7 @@
 *	https://www.weart.it/
 */
 
-#include "pch.h"
```

Comment:
I guess this will affect the visual studio compilation. To increase the change that upstream will merge, perhaps we can just wrap this lines with a `#ifdef` on a macro that we define with `target_add_definitions(..)` just in CMake, so that Visual Studio users are not affected?